### PR TITLE
Wheel changes to master

### PR DESCRIPTION
**Pull request by Vyacheslav-Smirnov**
- Update README
 - Set RPATH to dpctl and dpnp itself
 - Fix in setup.py

### CODE REVIEW

File: conda-recipe/build.sh [hunk 1]
```diff
@@ -25,6 +25,10 @@ if [ -n "${TBBROOT}" ]; then
     . ${TBBROOT}/env/vars.sh
 fi
 
+# Set RPATH for wheels
+export CFLAGS="-Wl,-rpath,\$ORIGIN/../dpctl,-rpath,\$ORIGIN $CFLAGS"
```
Comment (shssf) [inline conda-recipe/build.sh:29]: How it will work in Windows?

File: setup.py [hunk 2]
```diff
@@ -183,13 +180,13 @@
...
       long_description=__readme_file__,
+      long_description_content_type="text/markdown",
       author="Intel Corporation",
-      author_email="Intel Corporation",
       maintainer="Intel Corp.",
       maintainer_email="scripting@intel.com",
       url="https://intelpython.github.io/dpnp/",
       download_url="https://github.com/IntelPython/dpnp",
-      license=__license_file__,
+      license='BSD',
```
Comment (shssf) [inline setup.py:189]: Is this change related to "wheel testing"?

File: conda-recipe/build.sh (hunk 1 above)
Comment (Vyacheslav-Smirnov) [reply conda-recipe/build.sh:29]: The "build.sh" file is used only for unix build. For windows there should be bld.bat with build commands.
As there is no bld.bat file, windows build now is just fake)

File: setup.py (hunk 2 above)
Comment (Vyacheslav-Smirnov) [reply setup.py:189]: Yes, changing setup.py format is required to build a good wheel which can be uploaded to Pypi

**Comment by Vyacheslav-Smirnov** [comment]
@shssf , Are you ok with the changes?

**Comment by Vyacheslav-Smirnov** [comment]
Will be merged in scope of https://github.com/IntelPython/dpnp/pull/657
